backend/src/modules/azure/services/cron.azure.service.ts

```diff
@@ -0,0 +1,55 @@
+import { ConfigService } from '@nestjs/config';
+import { Cron } from '@nestjs/schedule';
+import axios from 'axios';
+import { Injectable } from '@nestjs/common';
+import {
+  AZURE_CLIENT_ID,
+  AZURE_CLIENT_SECRET,
+  AZURE_TENANT_ID,
+} from '../../../libs/constants/azure';
+import { CronAzureService } from '../interfaces/services/cron.azure.service.interface';
+
+@Injectable()
+export default class CronAzureServiceImpl implements CronAzureService {
+  constructor(private readonly configService: ConfigService) {
+    this.getAzureAccessToken();
+  }
+
+  private azureAccessToken: string | undefined = undefined;
+
+  @Cron('0 45 * * * *')
+  async handleCron() {
+    await this.getAzureAccessToken();
+  }
+
+  public getToken() {
+    return this.azureAccessToken;
+  }
+
+  getAzureAccessToken = async () => {
```

Comment:
Just to be more coherent 😉 

```suggestion
  async getAzureAccessToken() {
```